cmd/plugin/builder/command/cli_compile_test.go

```diff
@@ -94,3 +94,105 @@ func TestMergePluginManifest(t *testing.T) {
 		assert.Equal(foundPlugin.Versions, plugin.Versions)
 	}
 }
+
+func TestMergePluginGroupManifest(t *testing.T) {
+	assert := assert.New(t)
+
+	basePGManifest := cli.PluginGroupManifest{
+		Plugins: []cli.PluginNameTargetScopeVersion{
+			cli.PluginNameTargetScopeVersion{
```

Comment:
I'm getting a warning that you don't need to redeclare the type here.  The array declaration already specifies the type.  Similar warning in the test above this one.